### PR TITLE
ci: update labeler runner

### PR DESCRIPTION
**Pull request by luisan00**
### Contribution description
The **labeler check** should put `Area: doc` and `Area: ci` for  new PRs with changes in `md` files, in  the `.github` folder

### Testing procedure
- *labeler* shoild works as expected opening a new PR with changes in some of the `*.md` files in the `.github` folder

- or see the **labeler.yml** in `.github/`

### Issues/PRs references
depends on #202

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+- IMPORTANT! Document your contribution.
 -->
 
 
 ### Testing procedure
 
 <!--
-How should your contribution be tested? provide at least the following steps:
+### How should your contribution be tested? provide at least the following steps:
 - which test, example or piece of code need to be compiled
```
Comment (eduazocar) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: This is part of previous PR

Comment (luisan00) [reply .github/PULL_REQUEST_TEMPLATE.md]: for that the last line in my comment: **depends of** ..... my fail xD

**Review by eduazocar — COMMENTED** [review]
Everything looks good, go ahead :+1: ,  Do squash

**Review by eduazocar — DISMISSED** [review]
Ok let's go, merge it

**Pull request closed without merging**